src/azure-cli/azure/cli/command_modules/servicefabric/_params.py

```diff
@@ -311,7 +311,7 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('vm_size', help='The size of virtual machines in the pool. All virtual machines in a pool are the same size.', default='Standard_D2')
         c.argument('vm_image_publisher', help='The publisher of the Azure Virtual Machines Marketplace image.', default='MicrosoftWindowsServer')
         c.argument('vm_image_offer', help='The offer type of the Azure Virtual Machines Marketplace image.', default='WindowsServer')
-        c.argument('vm_image_sku', help='The SKU of the Azure Virtual Machines Marketplace image.', default='2019-Datacenter')
+        c.argument('vm_image_sku', help='The SKU of the Azure Virtual Machines Marketplace image.', default='2022-Datacenter')
```

Comment:
May I ask will this default value change cause a breaking change to customers?

Reply:
The parameter will only apply on creation so this should not be a breaking change to customers.

Reply:
Actually, what I want to confirm is whether it may cause a breaking change to the workflow of the customer's automation scripts. If there is a regression between `2022-Datacenter` and `2019-Datacenter`, then there is a risk of breaking change

Reply:
@a-santamaria can you help address this question.

Reply:
@zhagao-ms there shouldn't be issues with that. We are updating it since azure now recommends 2022 due to its security. In the future probably we would have to change it whenever there is a new version. What is the recommendation for this? is it better to remove the default?

Reply:
If you currently have evaluated that the change of default value has little impact on customers, then we can directly merge this PR.

Reply:
You can go ahead and merge